src/commands/strikes_command.py

```diff
@@ -0,0 +1,20 @@
+import discord
+from discord.ext.commands import Bot
+from util import is_user_moderator
+from enums import StrikeSeverity
+
+
+def create_strikes_commands(bot: Bot) -> None:
+    @bot.tree.command()
+    @discord.app_commands.check(is_user_moderator)
+    @discord.app_commands.describe(user="User being exiled")
```

Comment:
`User being striked`? not sure about the wording here @clairefleuret

Reply:
Why am I being mentioned

Reply:
Yeah being striked sounds fine